src/commcare_cloud/ansible/roles/postgresql_base/templates/postgresql.conf.j2

```diff
@@ -46,7 +46,11 @@ effective_io_concurrency = {{ postgresql_effective_io_concurrency }}
 # WRITE AHEAD LOG
 {% if postgresql_enable_archive and not is_pg_standby %}
 archive_mode = on
+{% if hot_standby_server %}
+archive_command = 'scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null %p {{hot_standby_server}}:{{ postgresql_archive_path }}/%f'
```

Comment:
why not just override `postgresql_archive_command` like you did with `postgresql_archive_path` instead of create a new branch here?

Reply:
can't override in `<env>public.yml` or in playbook (it will throw undefined var hot_standby_server) . so keeping it here.